js/script.js

```diff
@@ -1 +1,48 @@
-console.log('Hello, front end');
+// console.log('Hello, front end');
+
+
+var toCelsius = function (value) {
```

Comment:
Good use of separate functions for the temp conversion. These could also be put inside the "domContentLoaded" function as well if you didn't want them in the global scope.